WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -293,6 +293,30 @@ class RegisterDomainSuggestionsViewController: UIViewController {
         supportVC.showFromTabBar()
     }
 
+    // MARK: - Tracks
+
+    private func track(_ event: WPAnalyticsEvent, properties: [AnyHashable: Any] = [:], blog: Blog? = nil) {
+        var properties = properties
+
+        let defaultProperties = { () -> [AnyHashable: Any]? in
+            guard let blog else {
+                return nil
+            }
+            return WPAnalytics.domainsProperties(for: blog, origin: .allDomains)
```

Comment:
The original issue has regressed. `domains_dashboard_select_domain_tapped` is always tracked with `all_domains` as the origin, regardless of the source.

Flows checked:
* All Domains flow ✅ 
* Site Domains flow ❌ 
* Free to paid plan dashboard card ❌

Reply:
I was trying to refactor that "origin" param to be a string instead of a an enum with minimum impact on existing callers. I couldn't figure out a way to do that and I ended up forgetting about the original issue.

Thanks for the catch @hassaanelgarem. I'll try to resolve this today.

Reply:
Resolved in 7ec0de763a848ea249d456ccc38282f2e34357ac